README.md

```diff
@@ -231,92 +240,119 @@ curl \
 
-## Deployment <a name="app-deployment"></a>
+- Issues: Read-only
+  - Allows subscription to the "Issues" event
 
-### Manual deployment <a name="app-manual-deployment"></a>
+#### Organization permissions
 
-The whole application can be spawned with `docker-compose up`.
+- Projects: Read & write
+  - Allows for items to be created in
+    [GitHub Project](https://docs.github.com/en/issues/trying-out-the-new-projects-experience)s.
 
-For ad-hoc deployments, for instance in a VM, one idea is to use the
-`docker-compose up` command in a `tmux` session. e.g.
+#### Events subscriptions
 
-`tmux new -s github-issue-sync sh -c "docker-compose up 2>&1 | tee -a log.txt"`
+- Issues
+  - Events used to trigger syncing for our primary use-case
+
+### Installation <a name="app-github-app-installation"></a>
 
-## Dependencies <a name="app-dependencies"></a>
+Having [created](#app-github-app) and [configured](#app-github-app-configuration) the
+GitHub App, install it in a repository through
+`https://github.com/settings/apps/${APP}/installations`.
+
+## Setup <a name="app-setup"></a>
+
+### Requirements <a name="app-setup-requirements"></a>
 
 - `Node.js` for running the application
 - `yarn` for installing packages and starting scripts
+  - If it's not already be bundled with Node.js, install with
+    `npm install -g yarn`
 - `jq` for the filtering expressions on [Rules](#app-api-create-rule)
 - `postgres` for the database
-  - For local development we recommend
-    [running a Postgres instance through `docker`](#database-container)
+    See <https://gitlab.parity.io/groups/parity/opstooling/-/wikis/Setup#postgres>
 
-## Settings <a name="app-settings"></a>
+### Environment variables <a name="app-setup-environment-variables"></a>
 
-The permissions and event subscriptions can be configured at
-`https://github.com/settings/apps/${app}/permissions`
+All environment variables are documented in the
+[src/server/.env.example.cjs](./src/server/.env.example.cjs) file. For
+development you're welcome to copy that file to `src/server.env.cjs` so that all
+values will be loaded automatically once the application starts.
 
-### Repository permissions
+## Development <a name="app-development"></a>
 
-- Issues: Read-only
-  - Allows subscription to the "Issues" event
+### Run the application <a name="app-development-run"></a>
 
-### Organization permissions
+1. [Set up the GitHub App](#app-github-app)
+2. [Set up the application](#app-setup)
 
-- Projects: Read & write
-  - Allows for items to be created in
-    [GitHub Project](https://docs.github.com/en/issues/trying-out-the-new-projects-experience)s.
+    During development it's handy to use a [smee.io](https://smee.io/) proxy,
+    through the `WEBHOOK_PROXY_URL` environment variable, for receiving GitHub
+    Webhook Events in your local server instance.
 
-### Events subscriptions
+3. Start the Postgres instance
 
-- Issues
-  - Events used to trigger syncing for our primary use-case
+    See
```

Comment:
Perhaps in this case it *would* be nice to refer to `docker-compose` to run a development database since there is a `docker-compose.yml` anyway, but I like this approach of linking to one document with generic 'run a single postgres container' too.

Reply:
Yes, I prefer to have something more uniform in relation to the other projects since we're using a similar stack for all of them. I only created the `docker-compose.yml` because I wanted to try it in the sandbox VM (https://github.com/paritytech/devops/issues/1421), but in the end we did not really use it.